content/knowledge_base/consensus/avalanche/concepts.md

```diff
@@ -0,0 +1,35 @@
+# Consensus algorithm concepts behind Avalanche
+
+## Snow Algorithm group of protocols
```

Comment:
```suggestion
## Snow Algorithm Group of Protocols
```

Reply:
Changed section names to Chicago style here and everywhere else
https://github.com/optakt/optakt.github.io/pull/13/commits/211354094b1da3445d32b93c04cf950b6a935394#diff-7b8cdfdde84c7352f51f3cc642d28c90f9d1283870319c81956812353289fb32R3-R4